src/order-radix.c

```diff
@@ -4065,9 +4065,42 @@ int df_decreasing_expansion(SEXP x) {
 
 // -----------------------------------------------------------------------------
 
+/*
+ * `na_value` -> `na_last` is originally parsed as:
```

Comment:
May be confusing to call this argument `na_last` since it doesn't behave like `order::na.last`. `na_largest` might be a better name for this boolean. No need for action now though.